woocommerce/woocommerce
# [Accessibility] Add focus indicator to select and text inputs on Cart/Checkout block pages

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #51838  .
Closes #51791 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install a block-based theme.
2. Make sure the Cart and Checkout pages use the block variation.
3. On the frontend, add a product to the Cart.
4. Got to the Cart page.
5. Select "Change address" in the "Cart Totals" section.
6. Focus on the select and text inputs and verify they have an outline.
7. Go to the Checkout page.
8. Focus on the select and text inputs on the Checkout page and verify they have an outline.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
